environment/commands/clean/index.js

```diff
@@ -1,7 +1,7 @@
+const {exec} = require('child_process');
+
 class CleanCommand {
-  static async execute() {
-    console.log("TODO (chefsale): Implement CleanCommand");
-  }
+  static execute() { exec('rm -rf ~/.rainbowup && pm2 kill'); }
```

Comment:
I cannot find where `pm2` is used in our codebase, I could only find its declaration in `package.json`.

Reply:
I think once nearbridge is started with pm2 this would kill it. If it's not it's a no-op so it's okay.